code/modules/mob/living/basic/lavaland/watcher/watcher_projectiles.dm

```diff
@@ -0,0 +1,37 @@
+/// Chilling projectile, hurts and slows you down
+/obj/projectile/temp/watcher
+	name = "chilling blast"
+	icon_state = "ice_2"
+	damage = 10
+	damage_type = BURN
+	armor_flag = ENERGY
+	temperature = -50
+
+/obj/projectile/temp/watcher/on_hit(mob/living/target, blocked = 0)
+	. = ..()
+	if (!isliving(target))
+		return
+	apply_status(target)
+
+/// Apply an additional on-hit effect
+/obj/projectile/temp/watcher/proc/apply_status(mob/living/target)
+	target.apply_status_effect(/datum/status_effect/freezing_blast)
+
+/// Lava projectile, ignites you
+/obj/projectile/temp/watcher/magmawing
+	name = "scorching blast"
+	icon_state = "lava"
+	damage = 5
+	temperature = 200
+
+/obj/projectile/temp/watcher/magmawing/apply_status(mob/living/target)
+	target.adjust_fire_stacks(0.1)
+	target.ignite_mob()
+
+/// Freezing projectile, freezes you
+/obj/projectile/temp/watcher/icewing
```

Comment:
also can you `snake_case` these subtypes. i don't see why it can't be `magma_wing`, looks cleaner to read